specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -24,158 +25,105 @@
 
 ## Summary
 
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
```

Comment:
nit: I still think the section description or expectation is useful to have, continuously, since we keep updating the document.